pkg/metric/daemon.go

```diff
@@ -105,9 +105,14 @@ func (d *Daemon) GetStage() int {
 	return common.StageEssential
 }
 
+func (d *Daemon) IsHealthy(ctx context.Context) (bool, error) {
```

Comment:
Do you want to define a default health check? Then you can just embed that and don't need to define it every time